src/VisualStudio/CSharp/Impl/CSharpVSResources.resx

```diff
@@ -600,4 +600,7 @@
   <data name="Prefer_switch_expression" xml:space="preserve">
     <value>Prefer switch expression</value>
   </data>
+  <data name="Automatically_show_completion_list_in_argument_lists" xml:space="preserve">
+    <value>Automatically show completion list in argument lists (experimental)</value>
```

Comment:
```suggestion
    <value>Automatically show completion list when typing arguments (experimental)</value>
```

Reply:
note: i'm not sure if mine is better.

Reply:
I'm not sure either. I've left it as-is. Easy to change later.